pkg/sql/opt/norm/select_funcs.go

```diff
@@ -39,7 +39,7 @@ func (c *CustomFuncs) MapSetOpFilterLeft(
 	filter *memo.FiltersItem, set *memo.SetPrivate,
 ) opt.ScalarExpr {
 	colMap := makeMapFromColLists(set.OutCols, set.LeftCols)
-	return c.MapFiltersItemCols(filter, colMap)
+	return c.RemapCols(filter.Condition, colMap)
```

Comment:
I like that it turns out we already had the callback function basically written. Nice that you noticed this.

Reply:
Ya it was a great feeling when I realized in `DuplicateScanPrivate` that we just needed to pass `c.RemapCols` as the callback.